requirements.txt

```diff
@@ -1,97 +1,18 @@
-asdf>=2.10.1
-asdf-astropy>=0.2.2
-asdf-coordinates-schemas>=0.1.0
-asdf-standard>=1.0.3
-asdf-transform-schemas>=0.3.0
-asdf-wcs-schemas>=0.1.1
-astrodbkit2>=0.5
-astropy>=5.2.1
-astroquery>=0.4.6
-asttokens>=2.0.5
-attrs>=22.2.0
-backcall>=0.2.0
-beautifulsoup4>=4.9.3
-bokeh>=3.0.2
-certifi>=2022.12.7
-cffi>=1.14.5
-chardet>=4.0.0
-click>=8.1.3
-contourpy>=1.0.6
-cryptography>=39.0.1
-decorator>=5.1.1
-executing>=0.8.3
-exceptiongroup>=1.1.1
-Flask>=2.2.3
-Flask-Cors>=3.0.10
-Flask-WTF>=1.1.1
-greenlet>=1.1.1
-gwcs>=0.18.2
-html5lib>=1.1
-idna>=2.10
-importlib-metadata>=5.1.0
-importlib-resources>=5.10.0
-iniconfig>=2.0.0
-ipython>=8.10.0
-ipython-genutils>=0.2.0
-itsdangerous>=2.1.2
-jedi>=0.18.1
-jeepney>=0.6.0
-Jinja2>=3.1.2
-jmespath>=1.0.1
-jsonschema>=3.2.0
-keyring>=23.0.1
-markdown2>=2.4.0
-MarkupSafe>=2.1.1
-matplotlib-inline>=0.1.6
-mkl-fft>=1.3.1
-mkl-random>=1.2.2
-mkl-service>=2.4.0
-ndcube>=2.0.3
-numpy>=1.22.4
-olefile>=0.46
-packaging>=23.0
-pandas>=1.5.2
-parso>=0.8.3
-pexpect>=4.8.0
-pickleshare>=0.7.5
-Pillow>=9.3.0
-pip>=22.2.2
-pluggy>=1.0.0,<2.0
-prompt-toolkit>=3.0.30,<3.1.0
-ptyprocess>=0.7.0
-pure-eval>=0.2.2
-py>=1.11.0
-pycparser>=2.20
-pyerfa>=2.0.0.1
-Pygments>=2.11.2
-pyparsing>=3.0.9
-pyrsistent>=0.17.3
-pytest>=7.2.2
-python-dateutil>=2.8.2
-pytz>=2022.1
-pyvo>=1.4.1
-PyYAML>=6.0
-requests>=2.25.1
-scipy>=1.6.2
-SecretStorage>=3.3.1
-semantic-version>=2.8.5
-setuptools>=65.5.1,<67.3.0
-six>=1.16.0
-soupsieve>=2.2.1
-specutils>=1.9.1
-SPLAT>=0.32
-SQLAlchemy>=2.0.7
-stack-data>=0.2.0
-tomli>=2.0.1
-tornado>=6.2
-tqdm>=4.61.0
-traitlets>=5.1.1
-typing_extensions>=4.4.0
-urllib3>=1.26.5
-wcwidth>=0.2.5
-webencodings>=0.5.1
-Werkzeug>=2.2.3
-wheel>=0.38.1
-WTForms>=2.3.3
-xyzservices>=2022.9.0
-zipp>=3.4.1
+astrodbkit2>=0.5.1
+astropy>=5.2.0
+bokeh==3.0.2
+flask>=3.0.1
+flask-cors>=4.0.0
+flask-wtf>=1.2.1
+markdown2>=2.4.12
+multiprocess>=0.70.16
+numpy>=1.24.0
```

Comment:
We should provide an upper bound for the numpy version. Numpy 2.0 is coming soon and could break things.